api/home/DMT/job_handlers/azure_container_instances/__init__.py

```diff
@@ -0,0 +1,156 @@
+import logging
+from collections import namedtuple
+from time import sleep
+from typing import List, Tuple
```

Comment:
A fix for the job_handler bug is merged in DMT master. Should remove this temp workaround now

Reply:
resolved!